vanilla/counter-ts/src/index.ts

```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License (MIT)
  *
- * Copyright (c) 2019 Looker Data Sciences, Inc.
+ * Copyright (c) 2022 Looker Data Sciences, Inc.
```

Comment:
Interesting this update left the `*` characters are the start of the line but most of the other updates didn't

Reply:
Manually changed. Need to add linter rules to this. Another ticket though